flight/targets/coptercontrol/fw/pios_board.c

```diff
@@ -140,6 +140,10 @@ void PIOS_Board_Init(void) {
 	/* Mount all filesystems */
 	PIOS_FLASHFS_Logfs_Init(&pios_uavo_settings_fs_id, &flashfs_settings_cfg, FLASH_PARTITION_LABEL_SETTINGS);
 
+#if defined(ERASE_FLASH)
+	PIOS_FLASHFS_Format(pios_uavo_settings_fs_id);
+#endif
+
```

Comment:
Well we can remove this whole thing from F3/F4 since they have partition manager available in the bootloader. I really don't like that some boards carry on running the firmware after erasing, we should make a point of this not being for regular running IMO which is what was originally done here.